package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "@optbot/template",
-  "version": "0.0.2",
-  "description": "template for starting a new package",
-  "main": "lib/start",
+  "name": "@optbot/eqpuller",
+  "version": "0.0.0",
+  "description": "maintain a list of active equities",
+  "main": "lib/norequire",
```

Comment:
The scripts in the "scripts" field contain .js at the end. We should be consistent. I think you should either append .js here or remove them from the "scripts" script names.